### ISSUE TITLE
calc and or show condition for "by you" and "by summon", "by summon with item/skill effect"

### ISSUE THREAD

**Issue by jl452** [PathOfBuildingCommunity/PathOfBuilding#4218]
### Check for duplicates

- [X] I've checked for duplicate issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues)

### Is your feature request related to a problem?

may be "condition" in "calcs"-"skill hit damage"-"effective dps mod"-"enemy modifiers"-"notes" must say like "Ignited **by you**" and "Ignited **by any**" ?

i have Combustion with my summon and was confused that the checkmark "enemy ignited" works both on the Combustion's ignite from summon and on the "Snowforged" ignite from me

idk can it calc separately Ignited by summon with Combustion and any Ignited by me

ps: for minions debuff Elemental Army Support: Minions from Supported Skills inflict Exposure on Hit, applying -10% to the Elemental Resistance matching
now calc use for it Elemental Mastery: Exposure you inflict applies at least -18% to the affected Resistance and Fire Mastery: Fire Exposure you inflict applies an extra -5% to Fire Resistance but i think it wrong (idk how to test it)

### Describe the solution you'd like

separate debuffs from minions like it is now for Skitterbots's shock+chill and Corpse Pact's shock+chill

1) add checkboxe like "Combustion affected ignite" (not used as "is ignited", only "have -10% to Fire Resistance")
2) add checkboxes "status from minion" for status what can be affected/modified from passives (and items/skills?) or autocalc it
3) add "notes" text like "Ignited by you"

idk, it's hard ticket

**Comment by Wires77** [comment]
Yeah, currently there is no concept of where a debuff comes from in PoB, and might require quite a bit of work to fix.  Since there are so few sources, it is a lower priority, but I'll link other related issues here for reference in case someone wants to look into a fix.
